src/lib/y2network/config_writer.rb

```diff
@@ -23,9 +23,9 @@ module ConfigWriter
     # @param source [Symbol] Source name (e.g., :sysconfig)
     # @return [#config] Configuration writer from {Y2Network::ConfigReader}
```

Comment:
```suggestion
     # @return [Y2Network::Sysconfig::ConfigWriter]
```

also note that Y2Network::Autoinst::ConfigWriter does not exist